DOCSP-31186 Add sections for connections (#243)
mongodb/docs-spark-connector: source/getting-started.txt

         .. include:: /scala/api.rst

Integrations
------------

> You can integrate Spark with third-party platforms to use the {+connector-long+} in various external platforms.

Reviewer: i: repetitive use of 'platforms'

```suggestion
The following sections describe some of the popular third-party platforms that you can
integrate Spark and the {+connector-long+} with.
```

Amazon EMR
~~~~~~~~~~

> Amazon EMR is a managed cluster platform that you can run big data frameworks such as Spark on. To install Spark on an EMR cluster, see

Reviewer: i: [such as](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#s:~:text=Use%20such%20as%20without%20a%20comma)

```suggestion
Amazon EMR is a managed cluster platform that you can use to run big data frameworks like Spark. To install Spark on an EMR cluster, see
```

`Getting Started with Amazon EMR <https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-gs.html>`__ in the AWS documentation.

Databricks
~~~~~~~~~~

Databricks is an analytics platform for building, deploying, and sharing enterprise-level data. To integrate the {+connector-long+} with Databricks,
see `MongoDB <https://docs.databricks.com/aws/en/connect/external-systems/mongodb>`__ in the Databricks documentation.

Docker
~~~~~~

> Docker is an open-source platform that helps developers build, share, and run applications in containers. The following steps guide you through the process of connecting to a Docker container
> and integrating the {+connector-long+} in Docker.
> 
> 1. To start Spark in a Docker container, see `Apache Spark <https://hub.docker.com/r/apache/spark#!>`__ in the Docker documentation and follow the steps provided. 
> #. See `Create a Local Atlas Deployment with Docker <https://www.mongodb.com/docs/atlas/cli/current/atlas-cli-deploy-docker/>`__ to deploy Atlas on Docker. 
> #. Select the appropriate language in the tabs under :ref:`Getting Started <pyspark-shell>` and follow the steps provided. 

Reviewer: i: 'The following steps' suggest this is going to be a procedure, which feels out of place with the rest of these sections. I would probably just break it into separate links (and the last step is hopefully already understood, so I would leave it out):

```suggestion
Docker is an open-source platform that helps developers build, share, and run applications in containers.

- To learn how to start Spark in a Docker container, see `Apache Spark <https://hub.docker.com/r/apache/spark#!>`__ in the Docker documentation.
- To learn how to deploy Atlas on Docker, see `Create a Local Atlas Deployment with Docker <https://www.mongodb.com/docs/atlas/cli/current/atlas-cli-deploy-docker/>`__.
```
